src/util/tags.ts

```diff
@@ -31,15 +32,23 @@ function transformWhitelist(whitelist: WhitelistData) {
 }
 
 /** Create a tagify select menu out of a JSON input element */
-function tagify(input: HTMLInputElement, options?: TagifyOptions): Tagify<TagRecord>;
-function tagify(input: HTMLInputElement | null, options?: TagifyOptions): Tagify<TagRecord> | null;
+function tagify(element: HTMLInputElement, options?: TagifyOptions): Tagify<TagRecord>;
+function tagify(element: HTMLTagifyTagsElement, options?: TagifyOptions): Tagify<TagRecord>;
 function tagify(
-    input: HTMLInputElement | null,
-    { whitelist, maxTags, enforceWhitelist = true }: TagifyOptions = {},
+    element: HTMLInputElement | HTMLTagifyTagsElement | null,
+    options?: TagifyOptions,
+): Tagify<TagRecord> | null;
+function tagify(
+    element: HTMLInputElement | HTMLTagifyTagsElement | null,
+    { whitelist, maxTags, enforceWhitelist = true, editTags }: TagifyOptions = {},
 ): Tagify<TagRecord> | null {
-    if (input?.hasAttribute("name") && input.dataset.dtype !== "JSON") {
-        throw ErrorPF2e("Usable only on input elements with JSON data-dtype");
-    } else if (!input) {
+    // Avoid importing the HTMLTagifyTraitsElement class that references the foundry API
```

Comment:
What does this comment mean?

Reply:
Importing the class (that extends `foundry.applications.elements.AbstractFormInputElement`) directly for an `instanceof` check breaks the pack builder because tsx tries to resolve the import from *somewhere*.

Reply:
Ah, so its not that the code leads to avoiding the import, its that its an alternative to importing. Got it.